src/cattr/gen.py

```diff
@@ -181,6 +183,17 @@ def make_dict_structure_fn(cl: Type, converter, **kwargs):
                 f"    res['{ian}'] = {struct_handler_name}(o['{kn}'], __c_t_{an})"
             )
     lines.append("    }")
+    if forbid_extra_fields:
+        post_lines.append("  allowed_fields = {")
```

Comment:
Wasn't sure how much of a (non-)goal it is to have tidy generated code.  For example, I could probably have written
```python
post_lines.append(f"  allowed_fields = {{a.name} for a in attrs:r}")
```
which is shorter and maybe a little cleaner from the point of view of this function, but would likely lead to a very long line in the generated code in many cases.